### DISCUSSION TITLE
Date is kind of glitchy

### DISCUSSION THREAD

**Discussion by SyusufPhys** [community/community#10663] [Projects and Issues]
Any body else find the date input is kind of glitchy?

It bugs out when im attempting to enter it through the gui and auto clears when i try to enter it manually.

**Comment by sureshjoshi** [comment]
Agreed. I've tried for about 5 minutes to set Feb 28th, 2022 as a due date - via the picker (which keeps disappearing), then manually (which also disappears), and then by setting today (which sometimes works) and then editing and changing one date component at a time

**Reply by effendiian** [reply]
Yes, able to reproduce this issue. Date picker will disappear even if mouse is moving and clicking in the date picker window area multiple times. Manual entry will also wipe what I'm typing as it is being input. Should probably be denounced...
